etl_tests/conftest.py

```diff
@@ -266,9 +265,23 @@ def document_of_anti_recommendation_article(
 
 
 @pytest.fixture(scope="session")
-def anti_recommendation_by_key_tuple(
+def anti_recommendation(
+    anti_recommendation_record_key: RecordKey,
+    document_of_anti_recommendation_article: Document,
+) -> AntiRecommendation:
+    """Return an AntiRecommendation NamedTuple."""
+
+    return AntiRecommendation(
+        key=anti_recommendation_record_key,
+        document=document_of_anti_recommendation_article,
+        similarity_score=0.82,
+    )
+
+
+@pytest.fixture(scope="session")
+def anti_recommendation_keys_by_key_tuple(
     record_key: RecordKey, anti_recommendation_record_key: RecordKey
 ) -> tuple[dict[RecordKey, tuple[RecordKey]], ...]:
-    """Return a tuple containing an anti_recommendation_by_key dict."""
+    """Return a tuple containing anti_recommendation_keys_by_key dictionaries."""
```

Comment:
Just call it `anti_recommendation_graph` ?

Reply:
Yes, I decided to go with `anti_recommendation_graph`. I will use a different name for the next graph asset the ETL produces.